README.md

```diff
@@ -1,2 +1,25 @@
-# opa
-A library of policy algorithms deployed as an agent to policy-enable any product or service
+# OPA: Open Policy Agent
+
+OPA is an open source project which can help policy enable any application.
+
+## What is Policy?
+
+Policy defines expected behavior in response to specific events within an application. The implementation of policy can vary greatly within and across applications, e.g, it may be maintained manually by administrators, documented on wiki pages, hardcoded in applications, exposed through configuration, etc. Often, these policies are not part of the application's core business logic.
```

Comment:
Can we try to differentiate the policy that people write in English (in PDFs, in Word docs) from policy written in a declarative language that computers can understand.